includes/Core/Modules/Module.php

```diff
@@ -132,19 +141,22 @@ abstract class Module {
 	 * @param User_Options   $user_options   Optional. User Option API instance. Default is a new instance.
 	 * @param Authentication $authentication Optional. Authentication instance. Default is a new instance.
 	 * @param Assets         $assets  Optional. Assets API instance. Default is a new instance.
+	 * @param Transients     $transients Optional. Transients instance. Default is a new instance.
 	 */
 	public function __construct(
 		Context $context,
 		Options $options = null,
 		User_Options $user_options = null,
 		Authentication $authentication = null,
-		Assets $assets = null
+		Assets $assets = null,
+		Transients $transients = null
```

Comment:
I'm not sure we need to add `Transients` as a constructor dependency here as well since the only dependency it takes is `Context` anyways. If we did, we'd also need to update the code in `Modules` which instantiates each `Module` class to pass it along. I don't that is this is really necessary yet and `Modules` doesn't have access to a `Transients` instance either so we'd end up needing to add it to the constructor there too.

Reply:
Yeah I thought we needed this for tests but looks like it's not needed here. Thanks for spotting!